File: sdkwork-ai-sdk-kotlin/src/main/kotlin/com/sdkwork/ai/MusicGenerationRequest.kt
package com.sdkwork.ai

data class MusicGenerationRequest(
    val model: String? = null,
    val prompt: String? = null,
    val duration: Int? = null,
    val format: String? = null,
    val n: Int? = null,
    val style: String? = null,
    val metadata: Map<String, String>? = null,
    val negativePrompt: String? = null,
    val responseFormat: String? = null,
    val referenceAudio: String? = null
)

File: sdkwork-ai-sdk-csharp/Models/MusicGenerationRequest.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace Ai.Models
{
    public class MusicGenerationRequest
    {
        public string? Model { get; set; }
        public string? Prompt { get; set; }
        public string? NegativePrompt { get; set; }
        public int? Duration { get; set; }
        public string? Format { get; set; }
        public int? N { get; set; }
        public string? ReferenceAudio { get; set; }
        public string? Style { get; set; }
        public Dictionary<string, string>? Metadata { get; set; }
        public string? ResponseFormat { get; set; }
    }
}
